src/patternfly/components/Form/docs/code.md

```diff
@@ -1,14 +1,10 @@
-## Overview
-
-Only if needed write a short description with implementation notes. Design and interaction notes are already written on the design specs, don't repeat information.
-
 ## Accessibility
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
 | `for` | `<label id="{for}">` |  Each `<label>` must have a `for` attribute that matches its form field id. **Required** |
```

Comment:
For the "applied to" column, I think you mean `<label for="{id}">` or something like that? Or could it just say `<label>`?

Reply:
Under the "Usage" table...
* `.pf-c-form__select` says "Initiates a form radio" - should be "Initiates a form select"
* The `.pf-m-required` class only applies to labels and inputs. Should that extend to textareas, selects, etc?
* The `.pf-m-error` class only applies to helper text and inputs. Should that apply to textareas, selects, etc, too?

Reply:
That makes sense to me, although we are going to be doing more work on states so that might be the time to update this.

Reply:
> Is for and id necessary for accessibility even when the form element is nested inside the label?

I have the same question, but when I look at the [documented technique from WCAG](https://www.w3.org/TR/WCAG20-TECHS/H44.html) for this, they show no examples of that case. I'd recommend to always have `for` and `id` attributes even when wrapping the `<input>` with the `<label>` element to be sure that the label is associated with the control for all assistive devices.

Reply:
Looks like inputs, when nested inside labels, are associated implicitly. Also, explicit labelling seems to be better supported for assistive technologies. I think @jgiardino's approach is valid here.
https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly